pytest.ini

```diff
@@ -0,0 +1,10 @@
+# pytest.ini
+[pytest]
+minversion = 6.0
+addopts = -ra -q
+testpaths =
+    tests/GalSim/tests/
+    tests/jax
+filterwarnings =
+    ignore::DeprecationWarning
+    
```

Comment:
There is something weird going on with the last line in this file. Perhaps we can add a .pre-commit just to fix the endings of file and avoid this issue going forward ?

Reply:
I'm not sure black would touch these .ini files... in any case,I've tried to fix this file

Reply:
hum doesnt go away if I try to add text at the end or reformat... if you know how to fix this sort of thihngs let m knw